app/api/api/views.py

```diff
@@ -413,6 +422,52 @@ def perform_destroy(self, instance):
         instance.delete()
 
 
+class StructuralMappingTableAPIView(APIView):
+    def post(self, request, *args, **kwargs):
```

Comment:
I still think this should be a POST endpoint, unless it's now clear why in the original code it might be necessary?